pmd-core/src/main/java/net/sourceforge/pmd/renderers/JsonRenderer.java

```diff
@@ -21,6 +21,12 @@
 public class JsonRenderer extends AbstractIncrementingRenderer {
     public static final String NAME = "json";
 
+    // TODO do we make this public? It would make it possible to write eg
```

Comment:
For now, we don't need it public. We don't write a reader, just a writer, so the only important thing is, that this formatVersion ends up in the serialized json file.